api/instance.go

```diff
@@ -104,6 +112,7 @@ func (a *API) UpdateInstance(w http.ResponseWriter, r *http.Request) error {
 		if err := i.UpdateConfig(a.db, params.BaseConfig); err != nil {
 			return internalServerError("Database error updating instance").WithInternalError(err)
 		}
+		i.BaseConfig.SMTP.Pass = "" // hide from response
```

Comment:
`SMTP.Pass` is still output by this handler if `params.BaseConfig` is nil.

Is there anywhere that we *want* to output the SMTP password to JSON? If not we could do `json:"-"` in the tags. 

Or, if outputting it is just a special case, we could wrap it in a new struct with special tags there and do `json:"-"` by default.

Reply:
good catch. Thank you!

I thought about the json tag, but it really seems to complicated